package.json

```diff
@@ -22,19 +24,29 @@
     "url": "https://github.com/internetarchive/bookreader/issues"
   },
   "homepage": "https://github.com/internetarchive/bookreader#readme",
-  "dependencies": {},
   "private": false,
   "devDependencies": {
     "@babel/core": "7.12.9",
     "@babel/preset-env": "7.12.7",
+    "@internetarchive/ia-activity-indicator": "0.0.1",
+    "@internetarchive/ia-icons": "1.2.2",
+    "@internetarchive/ia-menu-slider": "0.1.10",
+    "@internetarchive/ia-sharing-options": "0.1.2",
+    "@internetarchive/icon-bookmark": "1.1.3",
+    "@internetarchive/icon-collapse-sidebar": "1.1.0",
+    "@internetarchive/icon-edit-pencil": "1.1.5",
+    "@internetarchive/icon-magnify-minus": "1.2.3",
+    "@internetarchive/icon-magnify-plus": "1.2.3",
+    "@internetarchive/icon-search": "1.2.3",
+    "@internetarchive/modal-manager": "0.1.0",
```

Comment:
_Question_ How come these are all in `devDependencies`?

Reply:
Because BookReader doesn't export as a normal npm module; it exports built files. So everything is a dev dependency in that sense. We'll need to change this at some point as we transition to allowing BR to be imported via JS.

Reply:
Hmm I think that's going to be a problem integrating this into petabox and offshoot. By doing this, it's going to register all of these custom elements twice, once inside petabox, then again in this package, and you get an error if you register a custom element twice.

Reply:
+1 on that, @jbuckner.
the first phase is asset swap on PB, then creating a custom component to load in offshoot

Reply:
Just closing the loop here after our chat on Friday. Since these, along with `lit-html` and `lit-element` below, are all dependencies of the exported `module`, they should go into `dependencies` so consumers can properly install them.